src/quic.c

```diff
@@ -1051,8 +1057,9 @@ const WOLFSSL_EVP_CIPHER* wolfSSL_quic_get_hp(WOLFSSL* ssl)
             break;
 #endif
 #if !defined(NO_AES) && defined(HAVE_AESCCM) && defined(WOLFSSL_AES_128)
+        /* This has to be CTR. See comment above. */
```

Comment:
Doesn't the build macro need fixed above? `WOLFSSL_AES_COUNTER`.

Reply:
Yes it does.